modules/local/get_nanolyse_fasta.nf

```diff
@@ -5,7 +5,9 @@ params.options       = [:]
 def options          = initOptions(params.options)
 
 process GET_NANOLYSE_FASTA {
-    output:    
+    container "quay.io/biocontainers/wget:1.20.1"
```

Comment:
I wonder if we can also have singularity + conda options here? Should be basic enough, can copy the fetchngs pipeline [`SRA_FASTQ_FTP`](https://github.com/nf-core/fetchngs/blob/0c43cc787290d88c9127af8281bb69554d905012/modules/local/sra_fastq_ftp.nf#L7-L10) process.

Might also be nice to use `curl` instead of `wget` as it tends to be more ubiquitous across systems.

Reply:
I have changed the process to use `curl` instead of `wget` and copied the lines from fetchngs pipeline `SRA_FASTQ_FTP` process.